scripts/license-header.txt

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
If this was to be changed at all, would running the script cause it to update all license headers?